picard/album.py

```diff
@@ -552,6 +553,31 @@ def switch_release_version(self, mbid):
             self.tagger.albums[mbid] = self
             self.load(priority=True, refresh=True)
 
+    def update_metadata_images(self):
+        new_images = []
+        orig_images = []
+
+        def process_file_images(file):
+            for image in file.metadata.images:
+                if image not in new_images:
+                    new_images.append(image)
+            try:
+                for image in file.orig_metadata.images:
+                    if image not in orig_images:
+                        orig_images.append(image)
+            except AttributeError:
```

Comment:
Under which circumstances does this raise an AttributeError? Assuming that `file` is always a `File`, it will always have `orig_metadata` and that will always have `images`.